plugins/woocommerce-admin/client/marketplace/components/constants.ts

```diff
@@ -11,3 +11,4 @@ export const MARKETPLACE_CART_PATH = MARKETPLACE_HOST + '/cart/';
 export const MARKETPLACE_COLLABORATION_PATH =
 	MARKETPLACE_HOST +
 	'/document/managing-woocommerce-com-subscriptions/#transfer-a-woocommerce-com-subscription';
+export const WP_ADMIN_PLUGIN_LIST_URL = '/wp-admin/plugins.php';
```

Comment:
Will this work with multisites? I remember [you warned me](https://github.com/woocommerce/woocommerce/pull/41066#discussion_r1379681088) for something similar in the past.

Reply:
Thanks, I missed that 🙂 
It's fixed now.